debian/changelog

```diff
@@ -1,3 +1,9 @@
+daos (2.3.103-2) unstable; urgency=medium
+  [ Michael Hennecke ]
+  * Change ipmctl requirement from v2 to v3
```

Comment:
should probably mention that libfabric requirement was also changed for accuracy